code/datums/mind.dm

```diff
@@ -143,7 +143,9 @@
 			martial_art.remove(current)
 		else
 			martial_art.teach(current)
-
+	if(isalien(new_character))
+		var/mob/living/carbon/alien/A = new_character
+		A.updateplasmadisplay()
```

Comment:
I'd rather this be done via signals instead, using the one send below, to avoid snowflaking species/mob specific stuff in here

Reply:
Done